app/Http/Middleware/VerifyAdminToken.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use App\Helper\Utils;
+use App\Models\User;
+use Closure;
+use Illuminate\Http\Request;
+
+class VerifyAdminToken
```

Comment:
since these two middlewares don't do anything with the token either, let's rename them too--something like `verify-circle-user`, `verify-circle-admin`

Reply:
updated